Chessington.GameEngine/Pieces/Knight.cs

```diff
@@ -10,7 +10,30 @@ public Knight(Player player)
 
         public override IEnumerable<Square> GetAvailableMoves(Board board)
         {
-            return Enumerable.Empty<Square>();
+            var currentPos = board.FindPiece(this);
+            var currentRow = currentPos.Row;
+            var currentCol = currentPos.Col;
+            List<Square> availableMoves = new List<Square>();
+
+            if (Square.At(currentRow - 2, currentCol + 1).IsInbound())
```

Comment:
Generally, use brackets for all if statements.
You should add all squares first and then use LINQ expression to filter the ones which are inbound. It will make your code more readable